workflow-templates/knative-go-build.yaml

```
@@ -59,5 +59,5 @@ jobs:
                 tr '\n' ' ')"
 
           echo "Building with tags: ${tags}"
-          go test -vet=off -tags "${tags}" -run=^$ ./... | grep -v "no test"
+          go test -vet=off -tags "${tags}" -run=^$ ./... | grep -v "no test" || true
```

I'd like to make sure this doesn't silently succeed.  Do we need `set -o pipefail`?

---

@dprotaso says this is the default

---

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference

the bash runner has pipefail set